frontend/src/components/BarChartWidget.tsx

```diff
@@ -198,7 +199,8 @@ const BarChartWidget = (props: Props) => {
                           TickFormatter.format(
                             tick,
                             xAxisLargestValue,
-                            props.significantDigitLabels
+                            props.significantDigitLabels,
```

Comment:
Changes from another PR should not be part of this one, you should create all the PRs branches from mainline.

Reply:
Thanks for letting me know Miguel, I've made changes to reflect this